src/drawing.cpp

```diff
@@ -238,7 +238,7 @@ uint16_t LocalDraw()
 
 // ShowStrip
 //
-// ShowStrip sends the data to the LED strip.  If its fewer than the size of the strip, we only send that many.
+// ShowStrip sends the data to the LED strip.  If it is fewer than the size of the strip, we only send that many.

```

Comment:
I think the originally intended spelling is "it's".

```suggestion
// ShowStrip sends the data to the LED strip.  If it's fewer than the size of the strip, we only send that many.
```

Reply:
What I'm saying is that Dave clearly intended to write "it's", in the meaning of the first bullet of on the page you linked to:

> If you mean it is or it has*, then use it’s

He just missed the apostrophe.